framework/packages/abstract-core/src/native/account_factory.rs

```diff
@@ -104,6 +104,8 @@ pub enum ExecuteMsg {
         namespace: Option<String>,
         // Provide list of module to install after account creation
         install_modules: Vec<ModuleInstallConfig>,
+        // Provide module salt, that will be added to instantiate2 salt of installed modules
+        module_salt: Option<Binary>,
```

Comment:
If also used as salt for account it should be renamed to `instantiate_salt` or similar. Preferably also renamed in the manager API.